nixos/tests/installed-tests/default.nix

```diff
@@ -48,7 +50,15 @@ let
               testConfig
             ] ++ optional withX11 ../common/x11.nix;
 
-            environment.systemPackages = with pkgs; [ gnome-desktop-testing ];
+            environment.systemPackages = [
+              (pkgs.writeShellScriptBin "nixos-installed-tests-runner" ''
+                runner() {
+                  ${pkgs.gnome-desktop-testing}/bin/gnome-desktop-testing-runner ${escapeShellArgs testRunnerFlags} -d '${tested.installedTests}/share' "$@"
```

Comment:
When debugging, I typically copy the `machine.succeed` invocation from the script printed on VM startup and then paste it into the REPL, tweaking it as needed. Having this path hidden in a shell package makes it harder to run this in interactive tests.